src/Tizen.NUI/src/internal/Interop/Interop.Texture.cs

```diff
@@ -46,6 +46,9 @@ internal static partial class Texture
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Texture_GetHeight")]
             public static extern uint Texture_GetHeight(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_Texture_TbmSurface")]
+            public static extern global::System.IntPtr Texture_New__TbmSurface(IntPtr jarg1);
```

Comment:
from next time, please change parameter "IntPtr jarg1" as a meaningful and intuitive named parameter~
example ) IntPtr tmbSurface 
Thank you~

Reply:
Ok. Thank you